### ISSUE TITLE
Min value range not working with percent

### ISSUE THREAD

**Issue by lorenzo-bonatti** [dm4t2/intl-number-input#20]
I'm trying to use the min range value with the format percent, but the element always returns the max value.
As you can see in the following screenshot in your playground, I inserted a minimum value of 10% and a maximum value of 50%, but the _Number value_ will always be 50%.

![image](https://github.com/dm4t2/intl-number-input/assets/47744162/c1df18fc-5253-4691-bccf-4330ed8744b9)
 
Update: if the min value is 0 it works correctly

**Comment by lorenzo-bonatti** [comment]
I think I found the problem.

In the `NumberInput` class there is the function `getMinValue` that uses the `getNextStep` function.

``` ts
private getMinValue(): number {
  let min = this.toFloat(-Number.MAX_SAFE_INTEGER)
  if (this.options.valueRange?.min !== undefined) {
    min = Math.max(this.options.valueRange?.min, this.toFloat(-Number.MAX_SAFE_INTEGER))
  }
  if (!this.options.autoSign && min < 0) {
    min = 0
  }
  min = this.getNextStep(min)
  return min
}

...

private getNextStep(value: number): number {
  return Math.ceil(value / this.step) * this.step
}
```

The function uses the `Math.ceil` function, which returns the next int value.
This creates a problem with percentage values that are inserted with decimals.

Here are some examples:
- min: 0.1, max: 0.5
The `getNextStep(0.1)` returns 1.
This is bigger than the max value and the NumberInput fixes the value to the max value (0.5)
- min: 1.2, max: 2.5
The `getNextStep(1.2)` returns 2.
The valid value is between 200% and 250%.

**Comment by dm4t2** [comment]
Hi, thanks for reporting. This is fixed in [v1.4.1](https://github.com/dm4t2/intl-number-input/releases/tag/v1.4.1) 🎉
